runtime/src/lib.rs

```diff
@@ -187,6 +191,19 @@ pub const fn deposit(items: u32, bytes: u32) -> Balance {
 /// A timestamp: milliseconds since the unix epoch.
 pub type Moment = u64;
 
+pub mod currency {
+	pub type Balance = u128;
+
+	pub const UNIT: Balance = 1_000_000_000_000;
+	pub const DOLLARS: Balance = UNIT; // 1_000_000_000_000
+	pub const CENTS: Balance = DOLLARS / 100; // 10_000_000_000
+	pub const MILLICENTS: Balance = CENTS / 1_000; // 10_000_000
+
+	/// The existential deposit.
+	pub const EXISTENTIAL_DEPOSIT: Balance = 10 * CENTS;
+}
+use crate::currency::DOLLARS;
```

Comment:
We use TEERs instead of dollars, see: https://github.com/integritee-network/integritee-node/blob/91d9ba56c402553e56c1d25f2923623cba3032fe/runtime/src/lib.rs#L179

So, you can delete this. If `DOLLARS` is needed below, you can replace the value with `TEER`